specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -16581,7 +19359,7 @@
           "type": "string"
         }
       },
-      "discriminator": "resourceType"
```

Comment:
it should be resourceType

Reply:
done

---
In reply to: [592121399](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r592121399) [](ancestors = 592121399)